exporter/jaeger/jaeger.go

```diff
@@ -14,18 +16,45 @@ func init() {
 	})
 }
 
+func getProcessTags(cfg *opencensus.Config) []jaeger.Tag {
```

Comment:
I'd rather return an error, so the function does not need to panic when something goes wrong

Reply:
I don't think that [silent discard](https://github.com/krakendio/krakend-opencensus/blob/master/opencensus.go#L69) of  Jaeger exporter functionality on parse failure is a good default behavior.

Reply:
I also think it's not good to discard invalid tag or to simply log an error and continue.

Reply:
WDYT? @kpacha